### PR TITLE
relxill

### PR DESCRIPTION
**Pull request by phajy**
I've added a folder `relxill`, the associated `Dockerfile`, and instructions for building the `relxill` local model into XSPEC. At the moment it is slightly hard-wired for `aarch64` architecture but hopefully any necessary changes are obvious to the user.

At the moment this is the only local model that we need to install, but looking to the future there might be more. However, for the moment I think it is fine to have a separate Docker image for each local model we're interested in - we can always consolidate them later.

### CODE REVIEW

File: relxill/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+RUN tar xfz relxill_tables.tgz
+RUN tar xfz relxill_model_v1.4.3.tgz
+
+RUN ./compile_relxill.sh
+
+WORKDIR /home/heasoft
+
+# Get XSPEC to automatically load the relxill model every time it starts
+RUN echo "lmod relxill /home/relxill" >> ~/.xspec/xspec.rc
```
Comment (fjebaker) [inline relxill/Dockerfile:29]: I would recommend adding
```dockerfile
CMD [ "fversion" ]
```
at the very end, as the entrypoint for the image. It's a bit of a flavour thing, but maybe you agree?

**Review by fjebaker — APPROVED** [review]

**Comment by fjebaker** [comment]
Looks good! If I'd tagged the HEASoft docker images correctly, there would have been an easy fix for getting the right architecture, but as it currently is I would need to consult the Docker documentation for manipulating `ARG`s during build time.

File: relxill/Dockerfile (hunk 1 above)
Comment (phajy) [reply relxill/Dockerfile:29]: Yes, sounds like a good plan - I'll do it!

**Comment by phajy** [comment]
I think I've updated the pull request with the `CMD` at the end and added a little bit to the `README.md`.

**Pull request merged**